src/components/Wallet.tsx

```diff
@@ -92,8 +92,16 @@ interface WalletProps extends RouteComponentProps {
   className: string
 }
 
+const id2Network = {
+  1: 'Mainnet',
+  3: 'Ropsten',
+  4: 'Rinkeby',
+  5: 'Goerli',
+  42: 'Kovan',
+}
```

Comment:
Would make sense to have this on `types.ts` together with the `Newtork` enum?

Reply:
:+1:

Reply:
On a second though, it's not a type. I'll move to utils